src/endpoint/s3/s3_rest.js

```diff
@@ -316,6 +318,42 @@ async function authorize_request_policy(req) {
     throw new S3Error(S3Error.AccessDenied);
 }
 
+async function authorize_request_iam_policy(req) {
+    const auth_token = req.object_sdk.get_auth_token();
+    const is_anon = !(auth_token && auth_token.access_key);
```

Comment:
Personally, I do not like name shortcuts
```suggestion
    const is_anonymous = !(auth_token && auth_token.access_key);
```

Reply:
I agree, I copied this from another function and wanted to use it as is.
Anyway, I changed it in the new code, thanks.